helm_drydock/drivers/node/__init__.py

```diff
@@ -13,16 +13,44 @@
 # limitations under the License.
```

Comment:
Same suggestion with the naming, perhaps `base.py`?

If I am also understanding the intention here, where these will be subclassed by more specific node driver classes, I would suggest this implement the full range of methods possible, returning raise NotImplemented exceptions.